Mathlib/RepresentationTheory/GroupCohomology/Resolution.lean

```diff
@@ -584,8 +584,7 @@ theorem forget₂ToModuleCatHomotopyEquiv_f_0_eq :
   · dsimp only [HomotopyEquiv.ofIso, compForgetAugmentedIso]
     simp only [Iso.symm_hom, eqToIso.inv, HomologicalComplex.eqToHom_f, eqToHom_refl]
   trans (linearCombination _ fun _ => (1 : k)).comp ((ModuleCat.free k).map (terminal.from _))
-  · dsimp
-    erw [Finsupp.lmapDomain_linearCombination (α := Fin 1 → G) (R := k) (α' := ⊤_ Type u)
+  · erw [Finsupp.lmapDomain_linearCombination (α := Fin 1 → G) (R := k) (α' := ⊤_ Type u)
```

Comment:
This seems to be an entirely unrelated change?

Reply:
This is necessary because the `free` no longer has the `simps` attribute.